src/Equations/damaged-elastic/Kernels/Time.cpp

```diff
@@ -703,3 +700,1491 @@ void seissol::kernels::Time::flopsTaylorExpansion(long long& nonZeroFlops,
 }
 
 unsigned int* seissol::kernels::Time::getDerivativesOffsets() { return m_derivativesOffsets; }
+
+void seissol::kernels::Time::calculateEps(
+    const real* exxNodal,
+    const real* eyyNodal,
+    const real* ezzNodal,
+    const real* exyNodal,
+    const real* eyzNodal,
+    const real* ezxNodal,
+    const unsigned int& q,
```

Comment:
Can also be passed by value.

Reply:
Modified, but is one preferred over the other? Is there an advantage of just passing by value here? I thought making copies is not preferred for memory.

Reply:
Have a look at https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in and consider https://godbolt.org/z/hTdK8edv5 (three vs two instructions).

Reply:
Okay, great. Makes sense! Thank you very much!